packages/core/package.json

```diff
@@ -45,12 +45,20 @@
         "default": "./dist/index.mjs"
       }
     },
+    "./macro": {
+      "require": {
```

Comment:
Don't we also need to specify the `"import"`?

Reply:
The current macro is build with unbuild with very specific settings. The declarations is turned of, and only code is produced. Since now macro is co-located with runtime code the build system should be configured accordingly.  One entry point produce typings and code, another one (macro) should not produce typings (we have the manually written) and should be copied as is. 

First of all this setup is very non-straightforward for generic build systems such as unbuild, so i was unable to configure it all. 

The workaround is not use unbuild for the macro entrypint and just expose it as-is. There only one hand-crafted cjs version. 
There is no sense in esm version in this case.  

The old macro was setup with esm version simply for consistency with other packages, there also no benefits of that.